### PR TITLE
#2048

### PR DESCRIPTION
**Pull request by jgauffin**
Added the possibility to remove volunteers from tasks.

Someone might want to check the text messages and suggest better texts.

### CODE REVIEW

File: AllReadyApp/Web-App/AllReady/Areas/Admin/Features/Tasks/RemoveVolunteerFromTaskCommandHandler.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            await _context.SaveChangesAsync();
+
+            if (!message.NotifyUser)
+                return;
+
+            var user = await _context.Users.FirstAsync(x => x.Id == message.UserId);
+            var task = await _context.VolunteerTasks.FirstAsync(x => x.Id == message.TaskId);
+            await _emailSender.SendEmailAsync(user.Email,
+                "Removed from a volunteer task",
+                $"The administrator have removed you from task '{task.Name}'.");
```
Comment (MisterJames) [inline AllReadyApp/Web-App/AllReady/Areas/Admin/Features/Tasks/RemoveVolunteerFromTaskCommandHandler.cs]: s/b: `$"The administrator has removed you from task '{task.Name}'.")`

**Comment by MisterJames** [comment]
Just a quick spelling fix and we should be good here. Thanks @jgauffin !

**Comment by jgauffin** [comment]
The spelling error is corrected :)

**Comment by MisterJames** [comment]
@jgauffin beauty, thanks!

**Pull request merged**